### DISCUSSION TITLE
How can I route notifications for my own repos to a specific email address?

### DISCUSSION THREAD

**Discussion by sketchbuch** [community/community#125553] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

An organisation that I want to join requires that my primary email address be the one they provide for me. This means all notifications, even for my own repos go to this email address.

How can I get all notifications about my own repos to go to a non-primary email address?

**Comment by judahpaul16** [comment]
Should be able to do this via: https://github.com/settings/notifications

![image](https://github.com/community/community/assets/23343173/43624999-9cdf-4fd9-aa6b-730e0e892393)
![image](https://github.com/community/community/assets/23343173/d4bba352-d674-4587-96f4-ca226a194c5b)

**Reply by sketchbuch** [reply]
Where in the drop down that is labelled "Pick organisation", do I get the option to pick my own repos? As I a said in my post:

> An organisation that I want to join requires that my primary email address be the one they provide for me. This means all notifications, even for my own repos go to this email address.

I'm not interested in routing an organisation's emails. I need to be able to route my notifications about my own repos to an email address other than my primary email address.
